airgap/api.go

```diff
@@ -100,6 +101,13 @@ type CallParams struct {
 	BlockNumber *big.Int
 }
 
+type FilterQueryParams struct {
+	Contracts []string        `json:"contracts"`
+	Topics    [][]common.Hash `json:"topics"`
+	FromBlock *big.Int        `json:"from_block"`
+	ToBlock   *big.Int        `json:"to_block"`
+}
```

Comment:
I believe you can specify a blockhash (mutually exclusive of from and to parameters) for a specific block which might be a nice option